geode-redis/src/main/java/org/apache/geode/redis/internal/Executor.java

```diff
@@ -27,9 +27,13 @@
    * @param command The command to be executed
    * @param context The execution context by which this command is to be executed
    */
-  void executeCommand(Command command, ExecutionHandlerContext context);
+  default void executeCommand(Command command, ExecutionHandlerContext context) {
+    executeCommandWithResponse(command, context);
+  }
 
-  default void execute(Command command, ExecutionHandlerContext context) {
+  default RedisResponse executeCommandWithResponse(Command command,
+      ExecutionHandlerContext context) {
     executeCommand(command, context);
+    return null;
```

Comment:
One other idea. I think the plan is to eventually have every executeCommand have a non-null response. But if we are going to continue to have some return "null" could they instead return an instance of RedisResponse to represents null?

Reply:
Will do once we switch the `PUBLISH` command which is the only one (I can think of) that would return null.